### ISSUE TITLE
Doesn't work for HTML or CSS for me

### ISSUE THREAD

**Issue by scriswell** [tpryan/Brackets-Reflow-Cleaner#3]
I'd love to work Reflow into my workflow and this extension could be the way to do it but sadly it doesn't do anything for me for HTML or CSS files.

Without this extension Reflow is not very useful at all. Too much recoding to justify the time spend in the editor for the prototype.

**Comment by aadamsx** [comment]
Same here -- HTML is fine but for CSS it does nothing. I'm on Rf 0.47.17127 and Brackets Sprint 41. Is this still an active project?

**Comment by pvbergen** [comment]
I encountered the same issue: HTML is working fine, but CSS is not fully executing.
Using the debugger mode in Brackets, I was able to implement a hacky workaround. It seems to enable the extension to do it's job.

The issue:
The input to the function this.hexToRGB on line 743 in in reflowCSSExtractor.js is not sanitized. The input variable hex may be undefined/null. 

Getting the workaround: 
The full version with the workaround is in my repository (https://github.com/pvbergen/Brackets-Reflow-Cleaner). I've made pull request, so Terrence may add this fix to this repository too.

The warning:
I have tested the change with a few css files. So far, it seems to work but I couldn't yet determine whether all the information is correctly extracted. Maybe my fix still needs improvement. At the moment I'm just content with having those 4000+ lines of Reflow CSS being reduced to a usable length.
